docs/sources/explore/trace-integration.md

```diff
@@ -57,11 +57,17 @@ Clicking anywhere on the span row shows span details.
 - Process metadata: Metadata about the process that logged this span.
 - Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations.
 
+##### Node graph
+
+You can optionally expand node graph for the displayed trace. Depending on the data source this can show spans of the trace as nodes in the graph, or some additional context like service map based on the current trace.
+
```

Comment:
```suggestion
You can optionally expand the node graph for the displayed trace. Depending on the data source, this can show spans of the trace as nodes in the graph, or as some additional context like service map based on the current trace.

```